group_vars/all.yml

```diff
@@ -50,10 +50,16 @@ kube_version: "latest"
 
 images_directory: /home/images
 virtual_machines:
-  - kube-master
-  - kube-node-1
-  - kube-node-2
-  - kube-node-3
+  - name: kube-master
+    role: master
```

Comment:
This tag here isn't related to ansible, it indicates the node's role in the cluster being stood up. node_type would work too, but not sure that it matters.

Reply:
I understand this isn't related to Ansible, but for readability of the code, I'd much prefer `node_type` here over an Ansible naming convention since it's a simple change to make.

Reply:
Sure, will change to node_type